dspace-server-webapp/src/test/java/org/dspace/app/rest/SitemapRestControllerIT.java

```diff
@@ -216,7 +217,13 @@ public void testSitemap_sitemapIndexXml() throws Exception {
                                       //** THEN **
                                       .andExpect(status().isOk())
                                       //We expect the content type to match
-                                      .andExpect(content().contentType("application/xml;charset=UTF-8"))
+                                      .andExpect(res -> {
+                                          String actual = res.getResponse().getContentType();
+                                          assertNotNull("Content type not set", actual);
```

Comment:
I don't understand why we needed this. Was there ever a concern that the result was null? We weren't testing this before.

Reply:
I think you want to know why we have to check `null`.  The method `content().contentType(<string>)`  do above checks by default. So I included that in custom ResultMatcher. If null check is not required, We may remove it.

Reply:
Yes, I was wondering because we weren't checking for null here before, and none of the other tests in this class check for null. My personal philosophy is that pull requests should be atomic and only fix the reported issue. I know that others feel less strongly about that though...

Reply:
OK great. You can remove the null check from both, as well as the extra import of `static org.junit.Assert.assertNotNull`. Thank you!